### PR TITLE
cap/samples reloaded 

### PR DESCRIPTION
**Pull request by danjoa**
Didn't find Rene...

### CODE REVIEW

**Comment by renejeglinsky** [comment]
Can't add myself

**Comment by danjoa** [comment]
Me neither... → you have to join https://github.com/sap-samples first, then we can add you to the team

**Comment by renejeglinsky** [comment]
I joined the team now. Thanks @danjoa @CHERIANS

File: orders/db/schema.cds [hunk 1]
```diff
@@ -0,0 +1,17 @@
+using { sap.capire.bookshop.Books } from '@capire/bookshop';
+using { Currency, managed, cuid } from '@sap/cds/common';
+namespace sap.capire.bookshop;
+
+entity Orders : cuid, managed {
+  OrderNo  : String @title:'Order Number'; //> readable key
+  Items    : Composition of many OrderItems on Items.parent = $self;
+  total    : Decimal(9,2) @readonly;
```
Comment (chgeo) [inline orders/db/schema.cds]: We don't have a good solution for the (dynamic) calculation of `Order.total`.
Shall we remove it for the time being?

Comment (chgeo) [reply orders/db/schema.cds]: @wattnu FYI

Comment (danjoa) [reply orders/db/schema.cds]: Yes, we should remove these fields for now

Comment (chgeo) [reply orders/db/schema.cds]: See #34

**Pull request merged**